source/_data/screencasts.yml

```diff
@@ -1,30 +1,21 @@
 # large links with thumbnails, links, embedded video
 large:
-  - title: "Testing Shadow DOM inside Custom Elements with Cypress"
-    url: https://www.youtube.com/watch?v=BsF6jer8Mxk
-    sourceName: Youtube
-    sourceUrl: https://www.youtube.com/watch?v=BsF6jer8Mxk
-    date: Oct 23, 2019
-    author: Adam Bien
-    youtubeId: BsF6jer8Mxk
-    slides: http://adambien.blog/roller/abien/entry/accessing_shadow_dom_insides_a
```

Comment:
removed because it's out of date now?

Reply:
Right, the workaround he demonstrates is no longer necessary.